packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```diff
@@ -177,8 +272,7 @@ popd
 %{_cross_libexecdir}/nvidia/tesla/bin/%{nvidia_tesla_470_version}/nvidia-smi
 
 # Libraries
-%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvidia-ml.so.1
-%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/libnvidia-ml.so.%{nvidia_tesla_470_version}
+%{_cross_libdir}/nvidia/tesla/%{nvidia_tesla_470_version}/*.so*
```

Comment:
This suggestion is part style, but mostly about future-proofing.

One concern I have with picking and choosing libraries in the `%install` section is that if new libraries appear in a driver update, we might not notice and then we'd ship without them.

An alternative that might help would be to bulk install all libraries, and create all the .so and .so.1 symlinks (if those are truly needed). Then down here in the `%files` section, group them by "compute", "graphics" etc, and exclude everything that we don't intend to ship.

That way, when we update the driver, the build will fail if a new library shows up, and we'll have to add it to the right section or exclude it as well.